CelesteNet.Client/Components/CelesteNetChatComponent.cs

```diff
@@ -349,9 +353,76 @@ public CelesteNetChatComponent(CelesteNetClientContext context, Game game)
             CurrentChannelName = tmp;
         }
 
+        #nullable enable
```

Comment:
I would probably just remove all the nullables from the code; ideally we would just enable them globally, but for now it's better if the codebase remains consistent

Reply:
I guess that makes sense.